tests/testthat/testEdgeThresh.R

```diff
@@ -0,0 +1,251 @@
+# edgeThresh.R
+#
+# Structure your tests according to the three principles expressed below:
+#    test_that("parameter errors are correctly handled", { ... })
+#    test_that("a sane input gives an expected output", { ... })
+#    test_that("a corrupt input does not lead to corrupted output", { ...})
+
+
+context("Graph permutation and delta calculation")
+
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+# OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+# logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+# NL <- .PlatformLineBreak()
+# if (file.exists(logName)) { file.remove(logName)}
+
+# ((Is rete.EGGprototype only the barebones valid EGG, or does it contain testable
+# data?))
+myEGG <- rete.EGGprototype
+nLargest <- max(igraph::components(EGG)$csize)
+permutedGraph <- .permuteGraph(EGG)
+nVertices <- igraph::vcount(EGG)
+nEdges <- igraph::ecount(EGG)
+# ==== END SETUP AND PREPARE ===================================================
+
+
+test_that("edgeThresh parameter errors are correctly handled", {
+    # Try each parameter missing in turn (only EGG has no defaults)
+    expect_error(edgeThresh())
+
+    # Try parameters out of expected bounds
```

Comment:
The convention that I have seen so far has been to separate tests for each parameter into sections and test one thing at a time in each test. Can you add comments for what is being tested in each expect_error test?